sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/CxxPlugin.java

```diff
@@ -367,16 +358,6 @@
       .onQualifiers(Qualifiers.PROJECT, Qualifiers.MODULE)
       .index(6)
       .build(),
-      PropertyDefinition.builder(CxxXunitSensor.PROVIDE_DETAILS_KEY)
```

Comment:
@jmecosta how is this with e.g Java. Is it possible to see the list with tests and jump to test code there?